### PR TITLE
Unifying indentation: python-for-beginners/chapter 8-10

### LINKED ISSUE #29: Unifying indentation: python-for-beginners/chapter 8-10
Unifying indentation to 4 spaces indent according to convention of #20, dealing with python-for-beginners/chapter 8-10

### PR DESCRIPTION
**Pull request by jesusleejy**
resolves #29

### CODE REVIEW

**Review by Dorika0203 — APPROVED** [review]
One commit is not related to the issue #29  - but I don't think that's quite a big problem.

File: python-for-beginners/08 - Handling conditions/code_challenge.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
...
-	tax = .07
-	print('Tax rate is: ' + str(tax))
+    tax = .07
+    print('Tax rate is: ' + str(tax))
 else
-	tax = 0
-print('Tax rate is: ' + str(tax))
\ No newline at end of file
+    tax = 0
+    print('Tax rate is: ' + str(tax))
```
Comment (devquint) [inline python-for-beginners/08 - Handling conditions/code_challenge.py]: This is not in `else` block

File: python-for-beginners/04 - String variables/format_strings.py [hunk 2]
```diff
@@ -4,5 +4,8 @@
 
 # the capitalize function will return the string with 
 # the first letter uppercase and the rest of the word lowercase
-print ('Hello ' + first_name.capitalize() + ' ' \
-       + last_name.capitalize())
+print (
+    'Hello ' + first_name.capitalize() +
+    ' ' +
+    last_name.capitalize()
+)
```
Comment (devquint) [inline python-for-beginners/04 - String variables/format_strings.py]: This changes should not be in this commit

**Review by devquint — CHANGES_REQUESTED** [review]
There's no other problem than I commented each file/line, check them please.

**Review by devquint — CHANGES_REQUESTED** [review]
I checked your 8-`code_challenge.py `.
However reverting commit by `revert` might cause some unpredictable output when we finally merge branches.

File: python-for-beginners/09 - Handling multiple conditions/or_statements.py [hunk 3]
```diff
@@ -1,10 +1,10 @@
 province = input("What province do you live in? ")
 tax = 0
 if province == 'Alberta' \
-   or province == 'Nunavut':
+    or province == 'Nunavut':
 	tax = 0.05
```
Comment (devquint) [inline python-for-beginners/09 - Handling multiple conditions/or_statements.py]: This still `\t`,
maybe make line 4 to 8 spaces and line 5 to 4 spaces would be good.

**Review by devquint — CHANGES_REQUESTED** [review]
I found your rebasing worked well.
However I found one `\t` missing, please check it.

**Review by devquint — APPROVED** [review]

**Review by Hyuntae-Jeong — APPROVED** [review]

**Pull request merged**